src/components/ImageSlider.jsx

```diff
@@ -31,16 +31,15 @@ export function ImageSlider({ images }) {
       <div
         className={`ease flex gap-2 transition-all duration-300 ${x_position[activeImg]}`}
       >
-      
         {images.map(({ src, title }, idx) => (
           <figure
+            key={idx}
```

Comment:
I like that you are using idx attribute from the map method to add keys for the images